src/services/info/notice/modules/NoticeList.vue

```diff
@@ -108,13 +107,16 @@ export default defineComponent<Props>({
             noticeItems: [] as NoticePostModel[],
             noticeItemTotalCount: 0,
             boardId: undefined as undefined | string,
+            searchText: undefined as undefined | string,
         });
 
         /* Api */
-        let noticeApiHelper = new ApiQueryHelper()
+        const initNoticeApiHelper = new ApiQueryHelper()
             .setPage(1, NOTICE_ITEM_LIMIT)
             .setMultiSort([{ key: 'options.is_pinned', desc: true }, { key: 'created_at', desc: true }]);
```

Comment:
이건 함수로 만드는 게 좋을 것 같아요! 혹시라도 `initNoticeApiHelper`를 직접 조작할 상황에 대비하여..

```suggestion
        const initNoticeApiHelper = (apiHelper: ApiQueryHelper) => {
          return apiHelper
              .setPage(1, NOTICE_ITEM_LIMIT)
              .setMultiSort([{ key: 'options.is_pinned', desc: true }, { key: 'created_at', desc: true }]);
        }
```

Reply:
아 근데 보니 noticeApiHelper를 여기에서도 조작해주네요...음 argument로 안 넣고 그냥 new ApiQueryHelper()...리턴하는 것도 괜춘할듯...
```
noticeApiHelper = getSearchFilter();
```

Reply:
아 네이밍이 좀 문제인듯 하네여.. initialNoticeApiHelper로 바꾸면 어떨까요??

Reply:
아 고런데, 아래에서 이렇게 할당해준 뒤, noticeApiHelper를 수정하면 initNoticeApiHelper는 안 바뀌는 거겠죠!?
긴가민가... 이거만 확인되면 함수형으로 만들 필요 없을듯요!
```
noticeApiHelper = initNoticeApiHelper;
```